actions/locations.js

```diff
@@ -47,7 +47,7 @@ function wrapper(work, location) {
         },
       });
 
-      dispatch(failed(location.key));
+      dispatch(failed(location.key, err));
```

Comment:
This would not be valid would it? We wouldn't have a term. Perhaps we should use something simpler.

Reply:
Yeah, you're right. I switched this to 'locations/unknown_error' for now. If there are common errors that we see in the future we can add in cases for them